addons/hw_escpos/controllers/main.py

```diff
@@ -29,9 +30,10 @@
     usb = None
 
 from odoo import http, _
-
 from odoo.addons.hw_proxy.controllers import main as hw_proxy
 
+from uuid import getnode as get_mac
```

Comment:
not needed, you can use `netifaces` to do the same
getnode may also [return a random value](https://docs.python.org/3/library/uuid.html#uuid.getnode) from time to time